content/solutions/altinn-studio/functional/ui-designer/components/index.md

```diff
@@ -107,13 +107,24 @@ weight: 100
         <img alt="Radio buttons component - default look" src="display-radio.png?width=700" title="Radio buttons component - default look" />
       </td>
     </tr>
+    <tr>
+      <td>Text Area</td>
+      <td>
+        <ul>
+          <li>Link to data model</li>
+          <li>Label text</li>
+          <li>Description text</li>
+          <li>Read only</li>
+      </td>
+      <td><img alt="Text area component - edit properties" src="props-text-area.png?width=700" title="Text area component - edit properties" /></td>
```

Comment:
The files has big .PNG but the img tag has reference to a file with smal .png so the pictures wont show